src/Passable.js

```diff
@@ -16,6 +15,9 @@ class Passable {
     enforce: Function;
 
     constructor(name: string, ...args) {
+        if (typeof name !== 'string') {
+            throw new TypeError(`[passable]: failed to execute 'Passable' constructor: Unexpected ${typeof name}, expected string`);
```

Comment:
Please note that as it has been suggested in a previous PR, we capitalize Passable in errors. It should then be:
````[Passable]: failed to execute 'Passable' constructor: Unexpected ${typeof name}, expected string````

Reply:
+1